lib/OpenQA/CacheService/Response.pm

```diff
@@ -5,8 +5,15 @@ package OpenQA::CacheService::Response;
 use Mojo::Base -base, -signatures;
 
 has [qw(data error)];
+
+# define soft-limit for inactive Minion jobs (enforced when determining idle worker slots availability)
 has max_inactive_jobs => $ENV{OPENQA_CACHE_MAX_INACTIVE_JOBS} // 5;
 
+# define hard-limit for inactive Minion jobs (enforced on setup of already started openQA job)
+has max_inactive_jobs_hard_limit => sub ($self) {
+    $ENV{OPENQA_CACHE_MAX_INACTIVE_JOBS_HARD_LIMIT} // ($self->max_inactive_jobs + 10);
```

Comment:
In https://progress.opensuse.org/issues/128276#Suggestions we suggested "cache_queue_limit_concurrent" = 3, "cache_queue_limit_max" = 50, so should we make the above 5 a 3 and then here make the 10 a 40?

Reply:
I don't think we should go as low as 3; also the limit of 5 is quite low. Making the hard limit a 40 would be ok, though.